packages/toggle/src/Toggle.tsx

```diff
@@ -23,13 +23,13 @@ const toggleInput = createDataProp('toggle-input');
 const toggleGroove = createDataProp('toggle-groove');
 
 const transitionInMS = 150;
-const focusRing = 3;
+export const focusRing = 3;
 
 const inputStyle = css`
   margin: 0;
   position: absolute;
-  left: 100%;
-  top: 100%;
+  left: 0;
+  top: 0;
```

Comment:
Just in case, let's add 
```css
height: 0;
width: 0;
margin: 0;
```

Reply:
we have margin but will add height and width